cmake/SFMLConfigDependencies.cmake.in

```diff
@@ -44,12 +48,17 @@ if(SFML_STATIC_LIBRARIES)
     # sfml-window
     list(FIND SFML_FIND_COMPONENTS "window" FIND_SFML_WINDOW_COMPONENT_INDEX)
     if(FIND_SFML_WINDOW_COMPONENT_INDEX GREATER -1)
-        if(FIND_SFML_OS_LINUX OR FIND_SFML_OS_FREEBSD)
+        if(FIND_SFML_OS_LINUX OR FIND_SFML_OS_FREEBSD NOT FIND_SFML_USE_DRM)
```

Comment:
```suggestion
        if(FIND_SFML_OS_LINUX AND NOT FIND_SFML_USE_DRM OR FIND_SFML_OS_FREEBSD)
```

Reply:
Oops! I fixed this in the v3 version of this commit which will come later, but since v2 has no install tests I forgot to fix it here. I just need to add an `AND` before `NOT FIND_SFML_USE_DRM`. What's the purpose of also rearranging the order of variables?

Reply:
I found a more expressive and succinct way to express this. I first check for DRM and if it's not DRM then we check for Linux or FreeBSD. I also actually tested this confirm that it works this time 😅